src/test/java/com/bringframework/reader/DefaultBeanDefinitionReaderTest.java

```diff
@@ -0,0 +1,50 @@
+package com.bringframework.reader;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.MockitoAnnotations.openMocks;
+
+import com.bringframework.registry.BeanDefinition;
+import com.bringframework.registry.BeanDefinitionRegistry;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+
+public class DefaultBeanDefinitionReaderTest {
```

Comment:
Junit 5 allows to have package private modifier as per class as per test methods , so lets remove public modifiers

Reply:
removed